libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java

```diff
@@ -1698,6 +1704,13 @@ public static String getEventNameForStat(AnalyticsTracker.Stat stat) {
                 return "unpublished_revision_dialog_load_local_version_clicked";
             case UNPUBLISHED_REVISION_DIALOG_LOAD_UNPUBLISHED_VERSION_CLICKED:
                 return "unpublished_revision_dialog_load_unpublished_version_clicked";
+            case POST_SIGNUP_INTERSTITIAL_SHOWN:
+                return "post_signup_interstitial_shown";
```

Comment:
Can we use a more generic name since it's used after signup and signin? I'm afraid this can be confusing for someone looking only at the analytics.

Reply:
I actually think the whole "post-signup interstitial" name could be confusing for anyone looking at this feature for the first time in the future. That being said, I agree it could be even _more_ confusing for someone looking only at analytics events. 

Got any suggestions for a new name? How about just `interstitial_shown`? Or should we have a more descriptive name like `welcome_interstitial_shown` or `no_sites_interstitial_shown`?

Reply:
We have decided to rename the events prefix from `post_signup_interstitial` to `welcome_no_sites_interstitial`.

Reply:
Just a short note: we picked `welcome_no_sites_interstitial` over `welcome_interstitial` because we thought we could add other welcome interstitial screens for other use cases in the future.